### ISSUE TITLE
Configure OAI endpoint

### ISSUE THREAD

**Issue by mzelesky** [pulibrary/marc_liberation#653]
If an OAI endpoint is configured, we can begin receiving incremental publishing from Alma in the sandbox.

**Comment by mzelesky** [comment]
https://developers.exlibrisgroup.com/alma/integrations/oai/

**Comment by kelynch** [comment]
I set up an OAI endpoint via the Integrations profile, largely following this - 

https://knowledge.exlibrisgroup.com/Alma/Product_Documentation/010Alma_Online_Help_(English)/090Integrations_with_External_Systems/030Resource_Management/060Setting_Up_OAI_Integration

I'm working on documentation for the values I used for the repository, they're test/sandbox values for what populates the Identify call, and will definitely change in production.

The OAI endpoint is IP restricted -- @kevinreiss do you know what suitable ranges we want to add so folks can get access to the endpoint?

**Comment by kevinreiss** [comment]
I'm not sure how to handle IP ranges considering GlobalProtect does not give one a stable public IP within Princeton IP space. Our app servers and the sonic wall VPN should be covered by 128.112.0.0 - 128.112.255.255.

**Comment by tpendragon** [comment]
I think we'd just give them the IPs of our indexing boxes for harvesting, unless you mean so we can index Alma locally for dev?

**Comment by kelynch** [comment]
We were able to confirm, adding the IPs of the machines that need to read the OAI-PMH feed in the Allowed IPs section of the OAI_PMH Integration profile allowed access.  We set up a Publishing Profile called "Incremental Dump" in the interface with the OAI protocol and ran a publishing job. 

Can we consider this issue resolved?  The endpoint will need to be enhanced when we turn to incremental publishing in the sandbox, but that seems like a separate task.
